hack/e2e.go

```diff
@@ -578,6 +571,131 @@ func (k kops) Down() error {
 	return finishRunning("kops delete", exec.Command(k.path, "delete", "cluster", k.cluster, "--yes"))
 }
 
+type KubernetesAnywhere struct {
```

Comment:
I tend to leave everything unexported until there's a cause to export it. Right now this doesn't matter because it's a `package main`, but if we shuffle things out to packages, it's a good place to start.